testing/test_config.py

```diff
@@ -1200,52 +1200,6 @@ def test_help_via_addopts(testdir):
     )
 
 
-def test_help_and_version_after_argument_error(testdir):
```

Comment:
Oh sorry I don't think it was clear: this test you should keep, I meant to remove the test you have added before (`test_help_and_version_verbose_after_argument_error`).

This test (and the adjustment you made to it) is necessary: when we get a parser error, we need to ensure we still display at least the pytest version.

I asked you to remove the "verbose" version of this same test because the output should actually be the same as the "non-verbose" version: the "verbose version" output includes plugins, which are not loaded when a parser error happens anyway, so we regardless fall back to the "non-verbose version" information.

Reply:
IOW: revert your dfcec18699e4031f36f5beaf4349886af7189fd5 commit and just delete `test_help_and_version_verbose_after_argument_error`, leaving `test_help_and_version_after_argument_error` as is. 👍 

Thanks!

Reply:
Gotcha! Will do.

Reply:
Hey there--so I checked this today and realized that I accidentally deleted the original test `test_help_and_version_after_argument_error` while trying to delete only`test_help_and_version_verbose_after_argument_error`. 

I'm really sorry for that confusing mistake, but I've pushed another commit to correct it.